atoma-bin/telemetry.rs

```diff
@@ -30,8 +30,8 @@ const NODE_LOG_FILE: &str = "atoma-node.log";
 const DAEMON_LOG_FILE: &str = "atoma-daemon.log";
 
 // Default Grafana OTLP endpoint if not specified in environment
-
-const DEFAULT_OTLP_ENDPOINT: &str = "http://localhost:4317";
+// Override this to be the localhost for local development if not using Docker
+const DEFAULT_OTLP_ENDPOINT: &str = "http://otel-collector:4317";
```

Comment:
I suggest not having this hardcoded here. It makes more sense to add it to the `config.toml`

Reply:
I actually disagree since we want to default to the docker instance and if one wants to override it then they can use an env var, but I feel this setup is more frictionless

Reply:
Makes sense